site/en/docs/privacy-sandbox/storage-partitioning/index.md

```diff
@@ -190,27 +190,20 @@ partitioned.
  
 ### Extension APIs
  
-[Extensions](/docs/extensions/mv3/) are programs
-that customize the browsing experience for the user. With Manifest V2,
-extensions can create
-[background pages](/docs/extensions/mv2/background_pages/)
-that have the extension's origin, but can embed iframes with web content's
-origins.
-
-Because partitioning the storage will break some use cases, a
-couple of mitigations will be provided. If the extension has
-[host_permissions](/docs/extensions/mv2/runtime_host_permissions/)
-for the iframe origin, then the iframe will be treated as the top-level frame
-and not the extension page. If an extension embeds an iframe with an extension
-URL into a top-level site with host permissions, the iframe will be treated as
-first-party with the extension rather than partitioned by the top-level site.
-
-{% Aside %} 
-Manifest V2 has been
-[deprecated](/docs/extensions/mv3/mv2-sunset/) and
-will be removed. It is recommended to
-[migrate to Manifest V3](/docs/extensions/mv3/intro/mv3-migration/).
-{% endAside %} 
+[Extensions](/docs/extensions/mv3/) are programs that allow users to customize their browsing
+experience.
+
+Extension pages (pages with a `chrome-extension://` scheme) can be embedded on sites
+across the web, and in these cases, they will continue to have access to their top-level partition.
+These pages can also embed other sites, in which case those sites will have access to their top-level
+partition as long as the extension has [host permissions](/docs/extensions/mv3/declare_permissions)
+for that site.
+
+In addition, extensions can use the [`chrome.cookies`](/docs/extensions/reference/cookies) API to
+interact with the cookies for a given site. This currently operates on cookies from all partitions
+in a way which can be surprising.
+
+For more information, see the [extension docs](/docs/extensions/mv3/storage-and-cookies).
```

Comment:
If all of these points are in storage-and-cookies, this should be cut down to something like "Extensions have their own storage partitioning rules. For information, see...

We should avoid duplicating content, particularly when the duplication is some place that will easily be overlooked by future members of our team.

Reply:
I think there's some benefit to providing a summary that developers can use to decide if it's worthwhile reading the rest of the documentation. I don't feel strongly though. Will defer to @rowan-m.

Reply:
I'm not saying don't have an example. I'm saying it needs to not be about MV2 migration. 

https://developers.google.com/style/timeless-documentation

Reply:
There are likely lots of extensions features where that argument can be made. The extensions team shouldn't have to worry about or keep track of references in far flung bits of documentation. I didn't say remove the link to our site and most, if not all, of the developers who need to know this are coming to this page _from_ the extensions site.